.ci/deploy-mac.sh

```diff
@@ -15,8 +15,31 @@ echo "AVVER=$AVVER" >> ../.ci/ci-vars.env
 cd bin
 mkdir "rpcs3.app/Contents/lib/"
 
-cp "/usr/local/opt/llvm@16/lib/c++/libc++abi.1.0.dylib" "rpcs3.app/Contents/lib/libc++abi.1.dylib"
+# check if we are on ARM or x86_64
+if [ "$BUILDING_FOR" = "arm64" ]; then
+	cp "/opt/homebrew/opt/llvm@16/lib/c++/libc++abi.1.0.dylib" "rpcs3.app/Contents/lib/libc++abi.1.dylib"
+		
+# we have ffmpeg libs bundled in the app bundle in rpcs3.app/Contents/Frameworks , now we need to change the rpath to point to it
+# this is needed because we are using a homebrewed ffmpeg for arm64, and the rpath points to the system ffmpeg
+# for example we want to point to rpcs3.app/Contents/Frameworks/libavcodec.60.dylib
+	install_name_tool -change "/opt/homebrew/Cellar/ffmpeg/6.0/lib/libavcodec.60.dylib" "@executable_path/../Frameworks/libavcodec.60.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
+	install_name_tool -change "/opt/homebrew/Cellar/ffmpeg/6.0/lib/libavdevice.60.dylib" "@executable_path/../Frameworks/libavdevice.60.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
+	install_name_tool -change "/opt/homebrew/Cellar/ffmpeg/6.0/lib/libavfilter.7.dylib" "@executable_path/../Frameworks/libavfilter.7.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
+	install_name_tool -change "/opt/homebrew/Cellar/ffmpeg/6.0/lib/libavformat.60.dylib" "@executable_path/../Frameworks/libavformat.60.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
+	install_name_tool -change "/opt/homebrew/Cellar/ffmpeg/6.0/lib/libavutil.58.dylib" "@executable_path/../Frameworks/libavutil.58.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
+	install_name_tool -change "/opt/homebrew/Cellar/ffmpeg/6.0/lib/libswresample.4.dylib" "@executable_path/../Frameworks/libswresample.4.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
+	install_name_tool -change "/opt/homebrew/Cellar/ffmpeg/6.0/lib/libswscale.7.dylib" "@executable_path/../Frameworks/libswscale.7.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
 
+else
+	cp "/usr/local/opt/llvm@16/lib/c++/libc++abi.1.0.dylib" "rpcs3.app/Contents/lib/libc++abi.1.dylib"
+
+	# change the path to app for sdl2
+	install_name_tool -change "/opt/homebrew/opt/sdl2/lib/libSDL2-2.0.0.dylib" "@executable_path/../Frameworks/libSDL2-2.0.0.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
```

Comment:
This is supposed to be x86
```suggestion
	install_name_tool -change "/usr/local/opt/sdl2/lib/libSDL2-2.0.0.dylib" "@executable_path/../Frameworks/libSDL2-2.0.0.dylib" "rpcs3.app/Contents/MacOS/rpcs3"
```